packages/nest/src/migrations/update-13-2-0/update-to-nest-8.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  formatFiles,
+  installPackagesTask,
+  readJson,
+  Tree,
+  updateJson,
+} from '@nrwl/devkit';
+import { sortObjectByKeys } from '@nrwl/tao/src/utils/object-sort';
+import { checkAndCleanWithSemver } from '@nrwl/workspace';
+import { satisfies } from 'semver';
+import {
+  nestJsSchematicsVersion8,
+  nestJsVersion8,
+  rxjsVersion7,
+} from '../../utils/versions';
+
+export default async function update(tree: Tree) {
+  const shouldUpdate = await isUpdatable(tree);
+
+  if (!shouldUpdate) {
+    return;
+  }
+
+  updateVersion(tree);
+
+  await formatFiles(tree);
+
+  return (): void => {
+    installPackagesTask(tree);
+  };
+}
+
+async function isUpdatable(tree: Tree) {
+  const json = readJson(tree, 'package.json');
+
+  if (json.dependencies['@angular/common']) {
+    const rxjs = checkAndCleanWithSemver('rxjs', json.dependencies['rxjs']);
+    if (satisfies(rxjs, rxjsVersion7)) {
+      return true;
+    } else {
+      const { Confirm } = require('enquirer');
+      const prompt = new Confirm({
+        name: 'question',
+        message: 'Do you want to update to RxJS 7 + Nest 8?',
+        initial: true,
+      });
+
+      return await prompt.run();
+    }
+  } else {
+    return true;
+  }
+}
```

Comment:
nitpick: same as above about `angular/common`

suggestion: let's clean up the `else` blocks in this check. If you return a value, you don't need `else`.

```suggestion
if (json.dependencies['@angular/common']) {
    const rxjs = checkAndCleanWithSemver('rxjs', json.dependencies['rxjs']);
    if (satisfies(rxjs, rxjsVersion7)) {
      return true;
    }

    const { Confirm } = require('enquirer');
    const prompt = new Confirm({
      name: 'question',
      message: 'Do you want to update to RxJS 7 + Nest 8?',
      initial: true,
    });

    return await prompt.run();
  }

  return true;
```